### ISSUE TITLE
Update docs with instructions on setup with Linux environments

### ISSUE THREAD

**Issue by adrianmcli** [UMAprotocol/protocol#1625]
For Ubuntu, here are the complete list of packages you'll need before running `npm install`:

```
make gcc libudev-dev g++ linux-headers-generic libusb-1.0-0
```

For Alpine:Linux:

```
libusb-dev eudev-dev linux-headers
```

**Comment by mrice32** [comment]
Sorry that list is only a partial list for alpine -- this should be the full list:
```
make gcc g++ libusb-dev eudev-dev linux-headers
```

**Comment by adrianmcli** [comment]
Closed in favour of #1645
